components/locale-chooser.js

```diff
@@ -2,47 +2,137 @@ import React from 'react'
   withContent,
 } from './locale'
+import EnSvg from './svgs/en.svg'
+import EsSvg from './svgs/es.svg'
+import FrSvg from './svgs/fr.svg'
 
-const Select = glamorous.select((props, {colors}) => ({
-  textAlignLast: 'center',
-  fontSize: 14,
-  height: 40,
+const Wrapper = glamorous.div({
+  fontSize: '.8em',
+  cursor: 'pointer',
+})
+
+const Toggle = glamorous.button((props, {colors, mediaQueries}) => ({
   backgroundColor: colors.white,
-  borderColor: colors.primary,
+  color: colors.primaryMed,
+  border: `1px solid ${colors.primaryMed}`,
+  textAlign: 'left',
+  padding: '3px 10px',
+  outline: 'none',
+  display: 'block',
+  fontSize: '1em',
+  width: '100%',
+  borderBottomColor: props.isOpen ? 'transparent' : colors.primaryMed,
+  [mediaQueries.smallOnly]: {
+    textAlign: 'center',
+  },
 }))
 
-export default withContent({component: 'locale-chooser'}, LocaleChooser)
+const List = glamorous.ul((props, {colors, mediaQueries}) => ({
+  display: 'flex',
+  position: 'absolute',
+  flexDirection: 'column',
+  padding: 0,
+  margin: 0,
+  opacity: '.9',
+  border: `1px solid ${colors.primaryMed}`,
+  [mediaQueries.smallOnly]: {
+    position: 'relative',
+    width: '100%',
+  },
+}))
+
+const Item = glamorous.li((props, {colors, mediaQueries}) => ({
+  display: 'flex',
+  flex: 1,
+  textAlign: 'left',
+  margin: 0,
+  backgroundColor: colors.white,
+  lineHeight: 1,
+  transition: 'all .3s',
+  '&:focus, &:hover, &:active': {
+    backgroundColor: colors.primaryMed,
+  },
+  [mediaQueries.smallOnly]: {
+    textAlign: 'center',
+  },
+}))
+
+const Link = glamorous.a((props, {colors}) => ({
+  width: '100%',
+  padding: '6px 10px',
+  transition: 'all .3s',
+  '&:focus, &:hover, &:active': {
+    textDecoration: 'none',
+    color: colors.white,
+  },
+}))
+
+const localeContent = ({key, display}) =>
+  (<div>
+    {getFlag(key)} <span>{display}</span>
+  </div>)
+
+const localeItem = ({key, display}) =>
+  (<Item key={key}>
+    <Link href={localeToHref(key)}>
+      {localeContent({key, display})}
+    </Link>
+  </Item>)
 
-function LocaleChooser({locale, content, ...rest}) {
-  return (
-    <Select
-      value={locale}
-      onChange={changeLanguage}
-      {...rest}
-      aria-label="Locale selector"
-    >
-      <option value="en">en</option>
-      <option value="es">es</option>
-      <option value="fr">fr</option>
-      <option value="help">{content.help}</option>
-    </Select>
-  )
+class LocaleChooser extends React.Component {
+  state = {
+    open: false,
+  }
+
+  handleClick() {
+    this.setState(prevState => {
+      return {open: !prevState.open}
+    })
+  }
+
+  render() {
+    return (
+      <Wrapper>
+        <Toggle onClick={this.handleClick.bind(this)} isOpen={this.state.open}>
+          {localeContent(locales.find(({key}) => key === this.props.locale))}
+        </Toggle>
+        {this.state.open &&
+          <List aria-label="Locale selector">
+            {locales.map(localeItem)}
```

Comment:
Is it OK to use `locales` from `locale.js`, because before the locales where hard coded.